js/__fixtures__/controllers/ed25519.json

```diff
@@ -0,0 +1,13 @@
+{
+    "@context": ["https://www.w3.org/ns/did/v1"],
```

Comment:
Can you locally cache this context as well?

Reply:
this a good catch, the document loader should be 100% explicit, and we should cache locally 100% of contexts.

Reply:
I have cached it locally (it was cached locally already, but better to be explicit with the document loader.